sos/plugins/networking.py

```diff
@@ -222,18 +222,10 @@ def setup(self):
                 ])
 
                 ss_cmd = ns_cmd_prefix + "ss -peaonmi"
-                ss_pred = SoSPredicate(self, kmods=['tcp_diag', 'udp_diag',
-                                                    'inet_diag', 'unix_diag',
-                                                    'netlink_diag',
-                                                    'af_packet_diag'])
```

Comment:
Good point to delete the duplicate code (cf. L157) - originally I thought it is an omission.